compiler/rustc_lint_defs/src/lib.rs

```diff
@@ -25,7 +25,11 @@ macro_rules! pluralize {
 /// before applying the suggestion.
 #[derive(Copy, Clone, Debug, PartialEq, Hash, Encodable, Decodable)]
 pub enum Applicability {
-    /// The suggestion is definitely what the user intended. This suggestion should be
+    /// The suggestion is definitely what the user intended, or maintains the exact meaning of the code.
+    /// This suggestion should be automatically applied.
+    ///
+    /// In case of multiple `MachineApplicable` suggestions (whether as part of
+    /// the same `multipart_suggestion` or not), all of them should be
     /// automatically applied.
```

Comment:
Hmm, is this true? People have wanted to do mutually exclusive suggestions in the past: https://discord.com/channels/442252698964721669/446773253953028107/651615006898782233

Reply:
This is true at present. We resolved that if we want to support multiple exclusive machine-applicable suggestions, we would have to add a new mechanism (see [this comment](https://github.com/rust-lang/rust/issues/53934#issuecomment-831396123) for more details).